almalinux/Dockerfile

```
@@ -45,7 +53,11 @@ RUN dnf groupinstall -y "Development Tools" && \
         bzip2-devel \
         xz-devel \
         sqlite-devel \
-        xmlsec1-devel
+        xmlsec1-devel \
+        procps-ng htop less \
+        strace lsof file \
+        iotop iftop \
+        tcpdump bind-utils
```

```suggestion
        tcpdump bind-utils \
        gcc
```
The uwsgi package requires a C compiler to build from source. Currently, the build fails with Exception: `you need a C compiler to build uWSGI`. Adding gcc to the list of installed packages will resolve this issue and allow uwsgi to compile successfully.